src/ipc/local/LocalIPCServiceAdapter.h

```diff
@@ -91,6 +91,12 @@ class LocalIPCServiceAdapter : public LocalIPCServiceAdapterBase
         registerService(objectPath, static_cast<ServiceType *>(serverObject));  // TODO: get rid of that cast
     }
 
+    template<typename T>
+    const char* typeToSignature() const
```

Comment:
no need for a signature for local IPC

Reply:
The LocalIPCServiceAdapterBase.h has method "appendDBUSIntrospectionData" which was probabely introduced to treat the ServiceAdapter layer consistenly so is "typeToSignature".